hack/images/dns/kube2sky/kube2sky.go

```diff
@@ -0,0 +1,292 @@
+/*
```

Comment:
well, upon further thought, either way the code is vendored. since it's not used in the main scheduler builds, it might as well be vendored here (that'll also make godep package updates easier).